src/components/ResortsData.js

```diff
@@ -63,7 +57,7 @@ class ResortsData extends Component {
             <table className="table table-striped">
               <ResortsListHead />
               <tbody>
-                {data.map(resort => <ResortBasicInfo resort={resort} />)}
+                {map(data, (resort) => <ResortBasicInfo resort={resort} />)}
```

Comment:
(minor)
We can use regular map, (not lodash one) as `data` is now array (not `byId` object).